tests/network/vmi_networking.go

```diff
@@ -213,54 +203,50 @@ var _ = SIGDescribe("[rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][level:c
 				Expect(job.WaitForJobToSucceed(remoteNodeTCPJob, 90*time.Second)).To(Succeed(), "should be able to reach VM workload from a pod on different node")
 			}
 		})
-
-		Context("VirtualMachineInstance with default interface model", func() {
-			BeforeEach(func() {
-				libnet.SkipWhenClusterNotSupportIpv4()
-
-				var err error
-				inboundVMI, err = virtClient.VirtualMachineInstance(testsuite.GetTestNamespace(nil)).Create(context.Background(), libvmifact.NewCirros(), metav1.CreateOptions{})
-				Expect(err).ToNot(HaveOccurred())
-				outboundVMI, err = virtClient.VirtualMachineInstance(testsuite.GetTestNamespace(nil)).Create(context.Background(), libvmifact.NewCirros(), metav1.CreateOptions{})
-				Expect(err).ToNot(HaveOccurred())
-
-				inboundVMI = libwait.WaitUntilVMIReady(inboundVMI, console.LoginToCirros)
-				outboundVMI = libwait.WaitUntilVMIReady(outboundVMI, console.LoginToCirros)
-			})
-
-			// Unless an explicit interface model is specified, the default interface model is virtio.
-			It("[test_id:1550]should expose the right device type to the guest", func() {
-				By("checking the device vendor in /sys/class")
-
-				// Taken from https://wiki.osdev.org/Virtio#Technical_Details
-				virtio_vid := "0x1af4"
-
-				for _, networkVMI := range []*v1.VirtualMachineInstance{inboundVMI, outboundVMI} {
-					// as defined in https://vendev.org/pci/ven_1af4/
-					checkNetworkVendor(networkVMI, virtio_vid)
-				}
-			})
-		})
 	})
 
-	Context("VirtualMachineInstance with custom interface model", func() {
+	Context("VirtualMachineInstance with custom and default interface models", func() {
 		It("[test_id:1770]should expose the right device type to the guest", func() {
 			By("checking the device vendor in /sys/class")
 			// Create a machine with e1000 interface model
 			// Use alpine because cirros dhcp client starts prematurely before link is ready
 			masqIface := libvmi.InterfaceDeviceWithMasqueradeBinding()
 			masqIface.Model = "e1000"
-			e1000VMI := libvmifact.NewAlpine(
+			masqIface.PciAddress = "0000:02:01.0"
+			const (
+				bridgeName = "br02"
+				nadName    = "simple-bridge"
+			)
+			bridgeIface := libvmi.InterfaceDeviceWithBridgeBinding(bridgeName)
+			bridgeIface.PciAddress = "0000:03:00.0"
+			bridgeNetwork := libvmi.MultusNetwork(bridgeName, nadName)
+			multiIfaceVMI := libvmifact.NewAlpine(
 				libvmi.WithInterface(masqIface),
 				libvmi.WithNetwork(v1.DefaultPodNetwork()),
+				libvmi.WithInterface(bridgeIface),
+				libvmi.WithNetwork(bridgeNetwork),
 			)
+			Expect(createBridgeNetworkAttachmentDefinition(testsuite.GetTestNamespace(nil), nadName, bridgeName)).To(Succeed())
```

Comment:
- This fits in a setup, not in the test itself.
- You cannot use a helper from a different test file. If there is a need to share a helper, it needs to go into `libnet` or just to repeat it. Please make sure not to use constants, variables, functions, etc.. from other test files.